Makefile

```diff
@@ -5,8 +5,8 @@ GMF_DIRECTIVES_PARTIALS_FILES := $(shell ls -1 contribs/gmf/src/directives/parti
 
 OS := $(shell uname)
 
-EXAMPLES_JS_FILES := $(shell find examples -maxdepth 1 -type f -name '*.js')
 EXAMPLES_HTML_FILES := $(shell find examples -maxdepth 1 -type f -name '*.html')
+EXAMPLES_JS_FILES := $(EXAMPLES_HTML_FILES:.html=.js)
```

Comment:
Yes, I do that because if I don't do that the https.js will be compiled and failed...